### PR TITLE
[WIP] Handle validation error

### PR DESCRIPTION
**Pull request by Marishka17**
### Motivation and context

|       Error         |CVATValidationError                          |ValidationError                         |
|----------------|-------------------------------|-----------------------------|
|some detail|`'some detail'`            |"[ErrorDetail(string='some detail', code='invalid')]"            |
|["some detail 1", "some detail 2"]          |`'some detail 1; some detail 2'`            |"[ErrorDetail(string='some detail 1', code='invalid'), ErrorDetail(string='some detail 2', code='invalid')]""            |
|{"key_1": "some detail 1", "key_2": "some detail 2"}          |`'key_1: some detail 1; key_2: some detail 2;'`|"{'key_1': ErrorDetail(string='some detail 1', code='invalid'), 'key_2': ErrorDetail(string='some detail 2', code='invalid')}"|
|{"key_1": ["some detail 1.1", "some detail 1.2"], "key_2": ["some detail 2.1", "some detail 2.2"]}          |`"key_1: ['some detail 1.1', 'some detail 1.2']; key_2: ['some detail 2.1', 'some detail 2.2'];"`|"{'key_1': [ErrorDetail(string='some detail 1.1', code='invalid'), ErrorDetail(string='some detail 1.2', code='invalid')], 'key_2': [ErrorDetail(string='some detail 2.1', code='invalid'), ErrorDetail(string='some detail 2.2', code='invalid')]}"|

### Checklist

- [x] I submit my changes into the `develop` branch
~~- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file~~
~~- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly~~
~~- [ ] I have added tests to cover my changes~~
~~- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
~~- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))~~

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by Marishka17** [comment]
@zhiltsov-max, Could you please take a look at this small PR?

**Comment by zhiltsov-max** [comment]
From the PR description I see that the errors are still strings, while they represent more complex structures. Is there a reason for this? Wouldn't it be better to pass json structures instead?

File: cvat/apps/engine/backup.py [hunk 1]
```diff
@@ -800,9 +802,7 @@ def export(db_instance, request):
                         try:
                             storage.upload_file(file_path, filename)
                         except (ValidationError, PermissionDenied, NotFound) as ex:
-                            msg = str(ex) if not isinstance(ex, ValidationError) else \
-                                '\n'.join([str(d) for d in ex.detail])
-                            return Response(data=msg, status=ex.status_code)
+                            return HttpResponse(str(ex), status=ex.status_code)
```
Comment (nmanovic) [inline cvat/apps/engine/backup.py:805]: What is the reason to use HttpResponse instead of Response?

**Comment by nmanovic** [comment]
@Marishka17 , could you please resolve the conflict?

**Pull request closed without merging**